packages/material-ui/src/Chip/chipClasses.d.ts

```diff
@@ -0,0 +1,45 @@
+export interface ChipClasses {
+  root: string;
+  sizeSmall: string;
+  sizeMedium: string;
```

Comment:
I just wanted to point out that I added the `medium` size for each size class. Is this correct to do? As they are not currently listed but I have seen them get added in other PRs

If so Ill add them to Chip.d.ts and generate the propType